src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -571,8 +573,9 @@ internal sealed class RSession : IRSession, IRCallbacks {
         /// <param name="url"></param>
         /// <returns></returns>
         Task IRCallbacks.WebBrowser(string url) {
```

Comment:
Ok, but if broker will be just a proxy, why can't we just return proxy url from RHost itself?

---
In reply to: [77257971](https://github.com/Microsoft/RTVS/pull/2242#discussion_r77257971) [](ancestors = 77257971)